Jinhee/routes/index.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express')
+const router = express.Router()
+
+const userRouter = require('./userRouter')
+router.use('/users', userRouter.router)
+
+const postRouter = require('./postRouter')
+router.use('/posts', postRouter.router)
+
+const likeRouter = require('./likeRouter')
+router.use('/likes', likeRouter.router)
```

Comment:
require 즉 다른 모듈을 Import 하는 코드들은 상단에 모아두는 것이 일반적입니다.
```suggestion
const userRouter = require('./userRouter')
const postRouter = require('./postRouter')
const likeRouter = require('./likeRouter')

router.use('/users', userRouter.router)
router.use('/posts', postRouter.router)
router.use('/likes', likeRouter.router)
```